dotCMS/src/main/java/com/dotmarketing/util/DateUtil.java

```diff
@@ -187,16 +191,47 @@ public static Date maxTime(final Date date) {
 	 */
 	public static Date convertDate(final String date,

```

Comment:
Can we add a convenience method in the DateUtil that just takes a string and trys all the date formats rather than having to pass all the date formats, e.g. the formats should be in here and not the ContentletAPI